assets/src/dashboard/app/views/templates/detail.js

```diff
@@ -99,6 +104,21 @@ function TemplateDetail() {
     });
   }, [templatesOrderById, templates]);
 
+  const relatedTemplates = useMemo(() => {
+    if (templatesOrderById && orderedTemplates) {
+      const getRandomTemplateId = () =>
+        templatesOrderById[
+          Math.floor(Math.random() * templatesOrderById.length)
+        ];
+      const randomlySelectedTemplates = [];
+      for (let i = 0; i < 5; i++) {
+        randomlySelectedTemplates.push(orderedTemplates[getRandomTemplateId()]);
+      }
+      return randomlySelectedTemplates;
+    }
+    return false;
```

Comment:
Style nit, in general, I feel types should stay consistent.  By its name and primary use `relatedTemplates` sounds like and is mostly an `array`.  I would expect that if there were no `relatedTemplates` that it would be set to an emtpy array (`[]`) instead of `false`.